src/website/app/demos/Text/examples/text.js

```diff
@@ -0,0 +1,19 @@
+/* @flow */
+import IconCloud from 'mineral-ui-icons/IconCloud';
+import Text from '../../../../../library/Text';
+
+export default {
+  id: 'basic',
+  title: 'Basic Usage',
+  description: `Text renders content with the correct styles and semantics.
```

Comment:
This should probably say something about "phrasing content". 

https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Content_categories#Phrasing_content

Reply:
I don't think this is true, though. Text can be any of those linked elements, but it can also be a `p` (default), `div`, `blockquote`, and many other non-phrasing elements.

Reply:
Fair enough, I guess that I was just trying to help users to create invalid DOM nesting.